eng/targets/Helix.props

```
@@ -35,4 +36,13 @@
     <HelixPreCommand Include="call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1 || exit /b 1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TestDependsOnNode)' == 'true' AND '$(IsWindowsHelixQueue)' == 'false'">
+    <HelixPreCommand Include="./installnode.sh $(NodeVersion) linux-x64" Condition="!$(HelixTargetQueue.Contains('OSX'))" />
+    <HelixPreCommand Include="./installnode.sh $(NodeVersion) darwin-x64" Condition="$(HelixTargetQueue.Contains('OSX'))" />
```

```suggestion
    <HelixPreCommand Include="./installnode.sh $(NodeVersion)" />
```
Because `installnode.sh` figures the platform out for itself.

---

I tried that but it doesn't work, OSX would resolve to linux-x64 instead of darwin-x64 and fail

---

Looks like $uname is undefined on the OSX.1014 queue